frontend/src/routes/manage/shared.tsx

```diff
@@ -491,30 +485,25 @@ const DEFAULT_SORT_DIRECTION = "DESCENDING";
 
 /** Helper functions to read URL query parameters and convert them into query variables */
 type QueryVars = {
-    limit: number;
-    offset: number;
+    page: number;
     direction: SortDirection;
 }
 export const parsePaginationAndDirection = (
     queryParams: URLSearchParams,
     defaultDirection: SortDirection = "DESCENDING",
 ): QueryVars => {
```

Comment:
I think it would be slightly nicer if this would return also `sortColumn: string`. That string can then be further parsed into the proper column by `createQueryParamsParser`. But I find it a bit weird that both functions call `split`.